test/randomized_benchmarking/test_randomized_benchmarking.py

```diff
@@ -345,6 +346,32 @@ def test_non_clifford_interleaved_element(self):
             lengths=lengths,
         )
 
+    def test_interleaving_delay(self):
+        """Test delay instruction can be interleaved."""
+        # See qiskit-experiments/#727 for details
+        interleaved_element = Delay(10, unit="us")
+        exp = rb.InterleavedRB(
+            interleaved_element,
+            qubits=[0],
+            lengths=[1],
+            num_samples=1,
+        )
+        # Not raises an error
+        _, int_circ = exp.circuits()
+
+        # barrier, clifford, barrier, "delay", barrier, ...
+        self.assertEqual(int_circ.data[3][0], interleaved_element)
+
+    def test_interleaving_circuit_with_delay(self):
```

Comment:
What is the difference of this test from above?

Reply:
It tests to interleave a circuit with delays, not a delay instruction.